flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerBaseTest.java

```diff
@@ -82,14 +82,16 @@ public void testKeyValueDeserializersSetIfMissing() throws Exception {
         new DummyFlinkKafkaProducer<>(
                 props, new KeyedSerializationSchemaWrapper<>(new SimpleStringSchema()), null);
 
-        assertTrue(props.containsKey(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG));
-        assertTrue(props.containsKey(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG));
-        assertTrue(
-                props.getProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG)
-                        .equals(ByteArraySerializer.class.getName()));
-        assertTrue(
-                props.getProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG)
-                        .equals(ByteArraySerializer.class.getName()));
+        assertThat(props).containsKey(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG);
+        assertThat(props).containsKey(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG);
+        assertThat(
+                        props.getProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG)
+                                .equals(ByteArraySerializer.class.getName()))
+                .isTrue();
+        assertThat(
+                        props.getProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG)
+                                .equals(ByteArraySerializer.class.getName()))
+                .isTrue();
```

Comment:
Is it expected to have these lines twice?

Reply:
Good catch,  looks like this was a broken test